usr/src/cmd/perl/Makefile.perl

```diff
@@ -26,6 +26,8 @@
 $(BUILDPERL64)PERLBINDIR = $(PERLDIR)/bin/$(MACH)
 $(BUILDPERL32)PERLBINDIR64 = $(PERLDIR)/bin/$(MACH64)
 
+$(CROSS_BLD)CERRWARN += -_gcc=-Wno-attributes -_gcc=-Wno-unused-function
```

Comment:
Is this because of the cross build, or the new perl version, or the aarch64 perl?  I'm just not sure if `$(CROSS_BLD)` is the thing we want to say, and whether we want to call it out with an `XXXARM` comment so we don't lose it later.

Reply:
you are right, we should for now probably put this under an `AARCH64_BLD` guard and put a comment on top